### PR TITLE
LUN-229: Fix retry exercise button

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/ResultsOverviewScreen.tsx [hunk 1]
```diff
@@ -117,6 +117,7 @@ interface ResultOverviewScreenPropsType {
 
 const ResultsOverview = ({ navigation, route }: ResultOverviewScreenPropsType): ReactElement => {
   const { exercise, results, discipline } = route.params.result
+  const { repeatExercise } = route.params
```
Comment (klinzo) [inline src/routes/ResultsOverviewScreen.tsx:120]: AFAIK functions should not be passed as route params, better to just pass the necessary values or just duplicate the function. This is also indicated by the warning: `Non-serializable values were found in the navigation state.` While it works, it is not good practice. See https://reactnavigation.org/docs/troubleshooting/#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state

**Comment by klinzo** [comment]
Tested on android

File: src/routes/ResultsOverviewScreen.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/routes/ResultsOverviewScreen.tsx:120]: I see, thank you. Wanted to avoid copying the function, but will do, in this case.

**Review by klinzo — APPROVED** [review]

**Review by f1sh1918 — APPROVED** [review]
Tested on android (emulator) & ios native :)

**Pull request merged**